graph-builder/src/config/settings.rs

```diff
@@ -37,11 +37,11 @@ pub struct AppSettings {
 
     // TODO(lucab): split this in (TLS, hostname+port).
```

Comment:
Not sure if this is still worth implementing, but this PR would be a good place for this

Reply:
In general I'd say no, the parser should handle non-TLS by now, see the test here: https://github.com/openshift/cincinnati/blob/f5a676548b8fb5d8c5218e7517ed97bf1b2408d0/graph-builder/src/registry.rs#L501-L507.